sematic/ui/src/types/Types.tsx

```diff
@@ -691,6 +684,18 @@ function LinkValueView(props: ValueViewProps) {
   );
 }
 
+function DatetimeValueView(props: ValueViewProps) {
+  const {valueSummary} = props;
+  const date = parseISO(valueSummary)
+
+  if (!valueSummary || !isValid(date)) {
+      return <Alert severity="error">Incorrect date value.</Alert>;
+  }
+  return (
+      <Typography>{format(date, "dd.MM.yyyy HH:mm")}</Typography>
```

Comment:
Yes, this is more appropriate. Let's format it this way maybe fore better readability:
```
November 6, 2022 8:06 PM UTC+2
```

Reply:
Changed the format in https://github.com/sematic-ai/sematic/pull/220/commits/13ecac698d3ef97025902e4d8999a2b17a17e489. Please note that I tried the format options from [date-fns](https://date-fns.org/v2.29.3/docs/format), and the available timezone formats were ISO-8601 and GMT, and I have chosen ISO-8601 time zone format, hope that is fine. 

Example:
<img width="834" alt="Screenshot 2022-11-12 at 11 50 05" src="https://user-images.githubusercontent.com/32076170/201471144-52a034d9-d46d-4131-8e54-1b094a88b054.png">